src/main/java/uk/gov/justice/digital/hmpps/whereabouts/services/court/CourtService.kt

```diff
@@ -33,11 +33,9 @@ class CourtService(
   private val videoLinkAppointmentRepository: VideoLinkAppointmentRepository,
   private val videoLinkBookingRepository: VideoLinkBookingRepository,
   private val clock: Clock,
-  private val eventStoreListener: VideoLinkBookingEventListener,
-  private val applicationInsightsEventListener: VideoLinkBookingEventListener,
+  private val videoLinkBookingEventListener: VideoLinkBookingEventListener,
```

Comment:
it doesn't matter too much, but I preferred keeping the EventStoreListener and AppInsightsListener as separate dependencies, as that guarantees the EventStore is updated before the AppInsights message is triggered